anesthetic/examples/utils.py

```diff
@@ -0,0 +1,63 @@
+"""Utility functions for nested sampling examples."""
+import numpy as np
+from scipy.stats import special_ortho_group
+from scipy.special import gamma, gammaln
+
+
+def random_ellipsoid(mean, cov, size=None):
+    """Draw a point uniformly in an ellipsoid.
+
+    This is defined so that the volume of the ellipsoid is sqrt(det(cov))*V_n.
+    where V_n is the volume of the unit n ball, and so that its axes have
+    length equal to the square root of the eigenvalues of the covariance
+    matrix.
+
+    Parameters
+    ----------
+    mean: 1d array-like
+        The center of mass of the ellipsoid
+
+    cov: 2d array-like
+        The covariance structure of the ellipsoid. Axes have lengths equal to
+        the square root of the eigenvalues of this matrix.
+
+    Returns
+    -------
+    points: array-like
+        Points drawn uniformly from the ellipsoid.
+        shape (*size, len(mean)).
```

Comment:
Why the star?

Reply:
You're right this is neither correct nor clear. I've updated in 62d2c17 the docstring so that it mirrors `np.random.multivariate_normal` which it's supposed to